edi/models/edi_exchange_record.py

```diff
@@ -147,3 +147,12 @@ def action_exchange_send(self):
         if not self.direction == "output":
             raise exceptions.UserError(_("An output record is required for sending!"))
         return self.backend_id.exchange_send(self)
+
+    def _exchange_processed_ok_msg(self):
```

Comment:
@etobella please check the new way to access this messages (no need for a new method every time)